human_clinical_and_health_data/data_sources.md

```diff
@@ -1,63 +1,145 @@
 
-### Considerations
+  The above can make it challenging to find and access human clinical and health data. Nonetheless, there are also commonalities in how data is organised and managed across Europe, which might help as a starting point to find and access the data:
+* **Electronic health records**: most European countries have implemented some form of electronic health records to store patient health data. These typically include patient demographics and medical information such as diagnoses, medications, treatment plans, immunisation dates, allergies, radiology images, and laboratory and test results. Electronic health records can be accessed by authorised healthcare professionals and are used to support clinical decision-making and continuity of care. For researchers outside of the health system it can be challenging to find and gain access to these records.
+* **Health data repositories**: in some countries, national or regional health data repositories have been established to aggregate electronic health records across multiple sources. For example, in Sweden, [the National Patient Register](https://www.socialstyrelsen.se/en/statistics-and-data/registers/national-patient-register/) collects data from all healthcare providers in the country; while in the UK, [the National Health Service (NHS) Digital](https://digital.nhs.uk/) collects data from various healthcare settings. However, in many countries these types of registries are not yet in place.
+* **Cross-border health data exchange**: the European Union is working to establish a common framework for cross-border exchange of health data: [The European Health Data Space (EHDS)](https://health.ec.europa.eu/ehealth-digital-health-and-care/european-health-data-space_en). This initiative aims to facilitate the exchange of health data across borders while ensuring data protection and privacy. The EHDS will include a set of technical standards and protocols to enable interoperability of health data across Europe. Currently the European Genome-phenome Archive (EGA) is expanding into a federation of nodes through the [Federated EGA](https://ega-archive.github.io/FEGA-onboarding/). FEGA enables deposition and distribution of health data that cannot be stored outside of the country of origin.
 
-Using a bullet point style list format as much as possible, describe the considerations for your topic that are specific to infectious diseases within the category (e.g. pathogen characterisation) that you have selected.
+## Finding (meta)data
```

Comment:
Do you think this section can be changed to "Search and discoverability" to align with other data sources pages?